cardscontainer.h

```diff
@@ -0,0 +1,34 @@
+#ifndef CARDSCONTAINER_H
```

Comment:
#pragma once

Reply:
qt samo tworzy sobie te ifndefy więc myślałem że powinniśmy to zostawić